### PR TITLE
Add full-url capture for sentry messages triggered in front end

### PR DESCRIPTION
**Pull request by OfekShilon**
I couldn't test this locally.  Is there a way to test in staging?

### CODE REVIEW

**Comment by mattgodbolt** [comment]
I'll try and test it locally

File: static/sentry.ts [hunk 1]
```diff
@@ -1,4 +1,3 @@
-// Copyright (c) 2023, Compiler Explorer Authors
```
Comment (mattgodbolt) [inline static/sentry.ts:1]: We should probably reinstate this

**Comment by mattgodbolt** [comment]
Looks like it works! Going to merge (after reinstating that line)

File: static/sentry.ts (hunk 1 above)
Comment (OfekShilon) [reply static/sentry.ts:1]: I think there's no need - the year is that of publishing 
https://www.copyrightlaws.com/copyright-symbol-notice-year/

Comment (mattgodbolt) [reply static/sentry.ts:1]: I meant...you deleted this line. It was there before and was OK, I was going to put it back as it's part of the rest of the codebase. Or am I missing something?

Comment (OfekShilon) [reply static/sentry.ts:1]: Oh, you're right of course. Sorry.

**Comment by partouf** [comment]
> ## Suspect Issues
> This pull request was deployed and Sentry observed the following issues:
> 
> * ‼️ **TypeError: r.map is not a function** `e.handleOutputButtonTitle(compiler-service)` [View Issue](https://compiler-explorer.sentry.io/issues/5494987673/?referrer=github-pr-bot)
> 
> Did you find this useful? React with a 👍 or 👎

This is only because the MSVC env returned a 502, so a result of MS servers being offline and not an actual issue because of this change

**Pull request merged**